pkg/api/constants/consts.go

```diff
@@ -20,4 +22,5 @@ const (
 	SessionClientHeaderValue     = "zot-ui"
 	APIKeysPrefix                = "zak_"
 	CallbackUIQueryParam         = "callback_ui"
+	APIKeyTimeFormat             = time.RFC1123Z
```

Comment:
let's decide what format we use for expiration date.

This format will be used by API clients to set expiration date on an API key.